src/inc/quic_platform.h

```diff
@@ -343,7 +343,8 @@ typedef enum CXPLAT_SELF_SIGN_CERT_TYPE {
 typedef enum CXPLAT_TEST_CERT_TYPE {
     CXPLAT_TEST_CERT_VALID_SERVER,
     CXPLAT_TEST_CERT_VALID_CLIENT,
-    CXPLAT_TEST_CERT_EXPIRED_SERVER
+    CXPLAT_TEST_CERT_EXPIRED_SERVER,
+    CXPLAT_TEST_CERT_EXPIRED_CLIENT
```

Comment:
The convention we've been going with (that apparently wasn't followed here before) is to always have a trailing `,`
```suggestion
    CXPLAT_TEST_CERT_EXPIRED_CLIENT,
```